talk/morelanguage/initialization.tex

```diff
@@ -98,8 +98,7 @@
     \end{cppcode*}
     \begin{itemize}
       \item If \mintinline{cpp}{T} has a default constructor (including compiler generated), calls it
-      \item Otherwise \mintinline{cpp}{t} is left uninitialized (undefined value)
-      \item If \mintinline{cpp}{T} is an array, default initializes all members
+      \item If \mintinline{cpp}{T} is a primitive type, \mintinline{cpp}{t} is left uninitialized (undefined value)
```

Comment:
```suggestion
      \item If \mintinline{cpp}{T} is a fundamental type, \mintinline{cpp}{t} is left uninitialized (undefined value)
      \item If \mintinline{cpp}{T} is an array, default initializes all members
```
Please keep the point on arrays!

Reply:
see the commit message, I think the about arrays is wrong? https://godbolt.org/z/6Paf5P49x

Reply:
Testing it further in godbolt, it looks like the correct statement is probably "If \mintinline{cpp}{T} is an array, same rules are applied to each item"

Reply:
fair enough :)

Reply:
suggestions applied

Reply:
oooh yeah now I get it, sorry :/